models/src/main/protobuf/CasperMessage.proto

```diff
@@ -238,9 +241,21 @@ message RChainState {
   int64         blockNumber  = 3;
 }
 
+message PaymentDeploy {
```

Comment:
There's too much info here.

There should just be an additional Par in the Deploy.
All of the other fields are available or derivable from the deploy.

Reply:
OK, I thought that for example `codeHash` would also be supplied by the client and validated in the node.

Reply:
Fixed @guardbotmk3 .